siriuspy/siriuspy/devices/dvf.py

```diff
@@ -251,3 +253,224 @@ def _set_and_wait(self, propty, value, timeout=None):
+    def roix_center(self):
+        """."""
+        return self['ImgROIXCenter-Mon']
+
+    @property
+    def roiy_fwhm(self):
+        """."""
+        return self['ImgROIYFWHM-Mon']
+
+    @property
+    def roix_fwhm(self):
+        """."""
+        return self['ImgROIXFWHM-Mon']
+
+    @property
+    def roiy_fit_amplitude(self):
+        """."""
+        return self['ImgROIYFitAmplitude-Mon']
+
+    @property
+    def roix_fit_amplitude(self):
+        """."""
+        return self['ImgROIXFitAmplitude-Mon']
+
+    @property
+    def roiy_fit_mean(self):
+        """."""
+        return self['ImgROIYFitMean-Mon']
+
+    @property
+    def roix_fit_mean(self):
+        """."""
+        return self['ImgROIXFitMean-Mon']
+
+    @property
+    def roiy_fit_sigma(self):
+        """."""
+        return self['ImgROIYFitSigma-Mon']
+
+    @property
+    def roix_fit_sigma(self):
+        """."""
+        return self['ImgROIXFitSigma-Mon']
+
+    @property
+    def roiy_fit_error(self):
+        """."""
+        return self['ImgROIYFitError-Mon']
+
+    @property
+    def roix_fit_error(self):
+        """."""
+        return self['ImgROIXFitError-Mon']
+
+    @property
+    def roiy_fwhm_factor(self):
+        """."""
+        return self['ImgROIYUpdateWithFWHMFactor-RB']
+
+    @roiy_fwhm_factor.setter
+    def roiy_fwhm_factor(self, value):
+        """."""
+        self['ImgROIYUpdateWithFWHMFactor-SP'] = value
+
+    @property
+    def roix_fwhm_factor(self):
+        """."""
+        return self['ImgROIXUpdateWithFWHMFactor-RB']
+
+    @roix_fwhm_factor.setter
+    def roix_fwhm_factor(self, value):
+        """."""
+        self['ImgROIXUpdateWithFWHMFactor-SP'] = value
+
+    @property
+    def roi_update_with_fwhm(self):
+        """."""
+        return self['ImgROIUpdateWithFWHM-Sts']
+
+    @property
+    def fit_angle(self):
+        """."""
+        return self['ImgFitAngle-Mon']
+
+    @property
+    def fit_sigma1(self):
+        """."""
+        return self['ImgFitSigma1-Mon']
+
+    @property
+    def fit_sigma2(self):
+        """."""
+        return self['ImgFitSigma2-Mon']
+
+    @property
+    def fit_proctime(self):
+        """Return image processing time [ms]."""
+        return self['ImgFitProcTime-Mon']
+
+    @property
+    def fit_angle_use_cmom_svd(self):
+        """."""
+        return self['ImgFitAngleUseCMomSVD-Sts']
+
+    @property
+    def log(self):
+        """."""
+        return self['ImgLog-Mon']
+
+    def create_image2dfit(self):
+        """Return a Image2DFit object with current image as data."""
+        imgfit2d = _Image2D_Fit(
+            data=self.image, fitgaussian=self._fitgaussian,
+            roix=self.roix, roiy=self.roiy)
+        return imgfit2d
+
+    def cmd_roi_update_with_fwhm(self, value):
+        """."""
+        self['ImgROIUpdateWithFWHM-Sel'] = bool(value)
+
+    def cmd_fit_angle_use_cmom_svd(self, value):
+        """."""
+        self['ImgFitAngleUseCMomSVD-Sel'] = bool(value)
```

Comment:
once these methods have an argument, maybe the more suitable names are:
`set_roi_update_with_fwhm`
`set_fit_angle_use_cmom_svd`

Reply:
I agree with @anacso17. @xresende, why not creating a property for these PVs?

Reply:
Today I agree with you guys but apparently not yesterday... lol. I was in a hurry yesterday and I did not realize it. I will change it.